native/cocos/platform/android/java/src/com/cocos/lib/CocosDownloader.java

```diff
@@ -157,6 +157,17 @@ public void run() {
                         if (tempFile.isDirectory()) break;
 
                         File parent = tempFile.getParentFile();
+                        if (parent == null) {
+                            String errStr = "Invalid path " + path + " : The current path is inaccessible, plz get writable path first";
```

Comment:
> plz get writable path first

Don't need the log. As
-  `getWritablePath` may be deprecated
- developer may use other function to get a writable path, such as the path in SD card.

Reply:
1